p2p/host/basic/basic_host.go

```diff
@@ -97,25 +93,14 @@ type HostOpts struct {
 
 	// NATManager takes care of setting NAT port mappings, and discovering external addresses.
 	// If omitted, this will simply be disabled.
-	NATManager NATManager
-
-	// BandwidthReporter is used for collecting aggregate metrics of the
-	// bandwidth used by various protocols.
-	BandwidthReporter metrics.Reporter
```

Comment:
Was only needed by the identity service because the old swarm didn't enable the bandwidth reporter early enough. That's no longer the case so the host doesn't need this now.